gemcalibration/include/gem/calib/CalibrationWeb.h

```diff
@@ -0,0 +1,91 @@
+/** @file CalibrationWeb.h */
+
+#ifndef GEM_GEMCAL_CALIBRATIONWEB_H
```

Comment:
header guards should match the namespace, i.e., `GEM_CALIB_` (also elsewhere)

Reply:
done